tests/bit_ops.rs

```diff
@@ -0,0 +1,424 @@
+
```

Comment:
You need to put the Apache license with Google copyright here at the top under Google CLA agreement. You can add a separate author: line below Copyright line if you want, see some examples at https://opensource.google/docs/copyright/#c